code/obj/item/implant.dm

```diff
@@ -1689,14 +1744,18 @@ circuitry. As a result neurotoxins can cause massive damage.<BR>
 /* ------------------------- Throwing Darts ---------------------- */
 /* =============================================================== */
 
-/obj/item/implant/projectile/bardart
+/obj/item/implant/projectile/dart/bardart
```

Comment:
In testing these didn't embed into humans- assuming that's a bug since there's code for the overlays

Reply:
Bar darts have a 5% chance to stick into you, which I left that way. Maybe that was the reason? Changing it to 100%, seemed to work fine for me. Not sure otherwise